services/storage/tests/test_datcore.py

```diff
@@ -35,6 +35,18 @@
     f = await dcw.list_files_recursively()
     assert len(f)
 
+async def test_datcore_list_files_raw(loop):
+    if not utils.has_datcore_tokens():
```

Comment:
FYI: try using [``@pytest.mark.skipif``](https://docs.pytest.org/en/latest/skipping.html#id1) ... it will lead to more informative logs in the tests